### PR TITLE
WorkerAssistant + TaskManager updates

### PR DESCRIPTION
**Pull request by markbulgakov**
Need it to transfer ALL fields from `sosw_tasks` to `sosw_closed_tasks`

### CODE REVIEW

File: sosw/managers/test/integration/test_task_i.py [hunk 1]
```diff
@@ -488,3 +498,10 @@ def test_get_average_labourer_duration__calculates_average(self):
         # Benchmarking
         # print(self.manager.get_stats())
         # self.assertTrue(False)
+
+
+    def test_get_task_by_id__(self):
```
Comment (nimirium) [inline sosw/managers/test/integration/test_task_i.py]: you could name this better :)

**Comment by nimirium** [comment]
Need it to transfer ALL fields from `sosw_tasks` to `sosw_closed_tasks`

**Comment by nimirium** [comment]
@ngr please take a look

**Comment by ngr** [comment]
The tests are broken. Please fix

**Pull request merged**